specification/Overview.md

```diff
@@ -9,18 +9,53 @@ organised by data category (saveframe).
 
 ### General issues
 
-  1. Residue and Atom names
-
-    Residues are identified by three strings ('chain_code', 'sequence_code',
-    'residue_type') that always are used together. Atoms are identified by a
-    residue identifier plus the atom name as a fourth string.
+  1. Requirements for NEF compatibility
+
+    The NEF format tries to support as many kinds of data as possible in a
+    simple file format. But we can clearly not require any program to
+    read and understand data that it does not normally support, be it complex or
+    crosslinked sequences, non-standard residues or residue variants, OR/AND
+    restraint logic, anonymous atom assignments, particular restraint types,
+    etc. If a program does support a certain type of data,
+    it is required to read and write such data correctly to/from NEF. If it
+    does not, it is free to skip such data, or convert them to something it can
+    deal with if this can be done appropriately.
```

Comment:
That would be a change to the current consensus, as I understand it. What you say is indeed desirable. But the way it was discussed, it was agreed that we could not require programs in general to keep and re-export data that they did not use. Let alone to interpret data that their programs are not equipped to handle, like non-stereospecifically assigned nuclei for programs that can not handle them. 
An inferior but more realistic solution would be to use a combination of the input file and output file for software B. As long as we are talking about program-specific fields, it should not be too hard to write a merge utility.